### DISCUSSION TITLE
What is useEffect?

### DISCUSSION THREAD

**Discussion by 0xdev99** [community/community#20749] [Discussions] [answered]
What is useEffect in React?
Please explain about it.

**Comment by dreamboat999** [comment, marked as answer]
useEffect(callback, dependencies) is the hook that manages the side-effects in functional components. callback argument is a function to put the side-effect logic. dependencies is a list of dependencies of your side-effect: being props or state values.
What does useEffect do? By using this Hook, you tell React that your component needs to do something after render. React will remember the function you passed (we'll refer to it as our “effect”), and call it later after performing the DOM updates.
